.github/workflows/amazon-cloudwatch-observability-image-scan.yaml

```diff
@@ -0,0 +1,84 @@
+name: Run Image Scan for Amazon CloudWatch Observability Helm Chart
+
+on:
+  schedule:
+    - cron: 0 13 * * 1 # Every Monday at 1PM UTC (9AM EST)
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  TERRAFORM_AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  AWS_DEFAULT_REGION: us-west-2
+
+jobs:
+  ContainerImageScan:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        container_images:
+          - registry: ".manager.image.repositoryDomainMap.public"
+            repository: ".manager.image.repository"
+            tag: ".manager.image.tag"
+
+          - registry: ".manager.autoInstrumentationImage.java.repositoryDomain"
+            repository: ".manager.autoInstrumentationImage.java.repository"
+            tag: ".manager.autoInstrumentationImage.java.tag"
+
+          - registry: ".manager.autoInstrumentationImage.python.repositoryDomain"
+            repository: ".manager.autoInstrumentationImage.python.repository"
+            tag: ".manager.autoInstrumentationImage.python.tag"
+
+          - registry: ".manager.autoInstrumentationImage.dotnet.repositoryDomain"
+            repository: ".manager.autoInstrumentationImage.dotnet.repository"
+            tag: ".manager.autoInstrumentationImage.dotnet.tag"
+
+          - registry: ".agent.image.repositoryDomainMap.public"
+            repository: ".agent.image.repository"
+            tag: ".agent.image.tag"
+
+          - registry: ".dcgmExporter.image.repositoryDomainMap.public"
+            repository: ".dcgmExporter.image.repository"
+            tag: ".dcgmExporter.image.tag"
+
+          - registry: ".neuronMonitor.image.repositoryDomainMap.public"
+            repository: ".neuronMonitor.image.repository"
+            tag: ".neuronMonitor.image.tag"
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure AWS Credentials
```

Comment:
nit: Do we still need the AWS credentials?

Reply:
seems like we shouldn't, but we will run into a rate exceeded error unless we authenticate:

https://github.com/aws-observability/helm-charts/actions/runs/10374721358/job/28722791358?pr=81